docs/src/rsh/errors/index.md

```diff
@@ -1043,6 +1043,26 @@ to be empty, but it wasn't. This issue may arise if there are statements beyond
 a `{!rsh} return` or `{!rsh} exit` statement. These statements are dead code
 and you can fix this issue by deleting them.
 
+```reach
+load: /hs/t/n/Err_Return_MustBeTail.rsh
+range: 13 - 20
+```
+
+The `{!rsh} return` statement on line 19 will never be executed. 
+The `{!rsh} return` statement in the `{!rsh} if ... else ...` block already handled what is needed for the `g` `{!rsh} function`.
+
+To fix the code, line 19 has to be removed.
+
+```reach
```

Comment:
This should be a file that is the correct version of `/hs/t/n/Err_Return_MustBeTail.rsh`
I'd name it `Fix_Return_MustBeTail.rsh`